packages/core/admin/admin/src/core/store/configure.ts

```diff
@@ -68,14 +68,23 @@ const configureStoreImpl = (
 ) => {
   const coreReducers = { ...staticReducers, ...injectedReducers } as const;
 
+  const defaultMiddlewareOptions = {} as any;
+
+  // These are already disabled in 'production' env but we also need to disable it in test environments
+  // However, we want to leave them on for development so any issues can still be caught
+  if (process.env.NODE_ENV === 'test') {
```

Comment:
This is the bit that I really hate because there shouldn't be checks in the codebase for test env, but I can't think of an alternative, and as I wrote in the comment it actually mirrors production. This feature only exists to catch issues in dev.

Reply:
This store configure function is only used in one test file – InjectReducer.test.tsx, should we just refactor that test instead? Then we shouldn't need this...

Reply:
Unfortunately it's also StrapiApp.tsx and a few others; pretty much anything that calls `new StrapiApp()` or `configureStore()`